src/cogs/summary.py

```
@@ -30,14 +31,43 @@ def __init__(self, bot: commands.Bot):
+                Locale.ja: "要約するメッセージの数。",
+            },
+            default=20,
+        ),
+        target_user: Member = SlashOption(
+            description="User to summarize messages from (optional).",
+            description_localizations={
+                Locale.zh_TW: "要總結訊息的目標使用者 (可選)。",
+                Locale.ja: "要約対象のユーザー (任意)。",
+            },
+            required=False,
+        ),
```

Ruff (0.8.2)

Do not perform function call `SlashOption` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)